include/dd/Definitions.hpp

```diff
@@ -82,5 +82,19 @@ namespace dd {
     //        return b;
     //    }
 
+    // calculates the Units in Last Place (ULP) distance of two floating point numbers
+    [[maybe_unused]] static std::size_t ulpDistance(dd::fp a, dd::fp b) {
+        if (a == b)
+            return 0;
+
+        std::size_t ulps = 1;
+        dd::fp      iter = std::nextafter(a, b);
```

Comment:
`iter` is not an iterator. Maybe use `nextFp` as variable name?

Reply:
Good suggestion! Changed it to `nextFP`.